src/script/scripting_mainmenu.cpp

```diff
@@ -41,6 +42,8 @@ MainMenuScripting::MainMenuScripting(GUIEngine* guiengine):
 
 	SCRIPTAPI_PRECHECKHEADER
 
+	initializeSecurity();
```

Comment:
So this will use the same whitelist as SSM? Maybe you could add a way to tell `initializeSecurity` not to create a backup of the insecure API since as far as I can see main menu doesn't need it?